backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/PluginsStorage.java

```diff
@@ -128,4 +128,22 @@ private List<File> getUnknownFiles() {
       return Collections.emptyList();
     }
   }
+
+  public void setReady() {
+    if(!Files.exists(pluginReferencesFilePath)) {
```

Comment:
Nitpick: Maybe a mall formatting issue here, should be `if (` with a space

Reply:
Thanks for the remainder. It seems I disabled it for some reason. I reenabled it.